backend-api/template.yaml

```diff
@@ -1112,6 +1112,126 @@ Resources:
         - !Ref PermissionsBoundary
         - !Ref AWS::NoValue
 
+  WellKnown5XXLowThresholdWarningAlarm:    
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      ActionsEnabled: true
+      AlarmDescription: "/.well-known/jwks.json endpoint has returned at least 2 5XX status codes twice in the past 5 minutes"
+      AlarmName: !Sub "${AWS::StackName}-LowThresholdWellKnown5XXApiGwAlarm"
+      AlarmActions:
+        - !Sub "arn:aws:sns:${AWS::Region}:${AWS::AccountId}:platform-alarms-sns-warning"
+      OKActions:
+        - !Sub "arn:aws:sns:${AWS::Region}:${AWS::AccountId}:platform-alarms-sns-warning"
+      ComparisonOperator: GreaterThanOrEqualToThreshold
+      DatapointsToAlarm: 2
+      EvaluationPeriods: 5
+      TreatMissingData: notBreaching
+      Threshold: 2
+      Metrics:
+        - Id: errorThreshold
+          Label: errorThreshold
+          ReturnData: true
+          Expression: IF((invocations>4) && (errorSum5XX>2) && (errorPercentage5XX>1),errorPercentage5XX,0)
```

Comment:
As discussed I think these should be >4

Reply:
I don't think we need this from the Expression `(errorSum5XX>2)` as this is calculated in line 1129 `(Threshold: 2)`

What do you think?

Reply:
My understanding is that the `threshold:2` should actually be `threshold:1` as this is the value that gets compared against the return value of expression which is `errorPercentage5XX` so I will rectify that.

However I think that `errorSum5XX` is also required because the criteria for this alarm is at least 2 5XX errors.

Which means in an example case of 10 invocations and 1 5XX (over 2 periods) error this would trigger the alarm as it would exceed the 1% threshold.